docx/document.py

```diff
@@ -100,6 +100,17 @@ def add_table(self, rows, cols, style=None):
         table.style = style
         return table
 
+    @property
+    def bookmarks(self):
+        """|Bookmarks| object providing access to |Bookmark| objects.
+
+        This feature is PARTIAL and EXPERIMENTAL. The current implementation
+        only provides access to bookmarks in the main document story.
+        Conflicts may arise if a bookmark is added with the same name as one
+        appearing in another story such as a header or footer etc.
+        """
+        return self._body.bookmarks
```

Comment:
Because of the multi-story scope of bookmarks, I'm inclined to delegate ownership of the actual Bookmarks object to _Body, like `.paragraphs` does with those. Eventually this `Document`-level bookmarks may become an "aggregate" object that contains bookmarks from all stories in the document; so this seam will allow us to add that later (and incrementally) with a minimum of disruption.